Backend/app/utils/knowledge_seeder.py

```
@@ -0,0 +1,127 @@
+import asyncio
+import logging
+import sys
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)